packages/feedback-service/src/resolver.ts

```diff
@@ -25,32 +24,86 @@ export const FeedbackResolver = {
             headers: { 'Content-Type': 'application/json' },
             body: graphql
           };
-          fetch('http://localhost:8080/graphql', requestOptions)
+          return fetch('http://localhost:8080/graphql', requestOptions)
```

Comment:
Use environment varables for the User Service here.
Do not commit development/debugging code to git.